pulumi/integration_tests/__main__.py

```diff
@@ -0,0 +1,121 @@
+import os
+import sys
+from pathlib import Path
+
+sys.path.insert(0, "..")
+
+import pulumi_aws as aws
+import pulumi_nomad as nomad
+from infra import config
+from infra.autotag import register_auto_tags
+from infra.nomad_job import NomadJob, NomadVars
+from infra.quiet_docker_build_output import quiet_docker_output
+
+import pulumi
+
+
+def main() -> None:
+    ##### Preamble
+
+    stack_name = stackname_sans_prefix()
+
+    quiet_docker_output()
+
+    # These tags will be added to all provisioned infrastructure
+    # objects.
+    register_auto_tags({"grapl deployment": stack_name})
+
+    if not config.LOCAL_GRAPL:
+        # Set the nomad address. This can be either set as nomad:address in the config to support ssm port forwarding or
+        # taken from the nomad stack
+        nomad_config = pulumi.Config("nomad")
+        nomad_override_address = nomad_config.get("address")
+        # We prefer nomad:address to support overriding in the case of ssm port forwarding
+        nomad_server_stack = pulumi.StackReference(f"grapl/nomad/{stack_name}")
```

Comment:
FYI, I've got a change that will DRY this logic up a bit in a PR

Reply:
cool! yeah, i was thinking of doing it myself